File: back-end/src/main/java/com/szefi/uml_conference/model/dto/socket/ActionTarget.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.szefi.uml_conference.model.dto.socket;

/**
 *
 * @author h9pbcl
 */
public class ActionTarget {
    private String target_id;
private String parent_id;
private String _type;


    public String getTarget_id() {
        return target_id;
    }

    public void setTarget_id(String target_id) {
        this.target_id = target_id;
    }

    public String getParent_id() {
        return parent_id;
    }

    public void setParent_id(String parent_id) {
        this.parent_id = parent_id;
    }

    public String getType() {
        return _type;
    }

    public void setType(String _type) {
        this._type = _type;
    }
}

File: front-end/src/app/components/models/socket/ActionTarget.ts
export class ActionTarget {
  constructor(id = '--', _type = '--', parent_id = '--') {
    this.target_id = id;
    this.parent_id = parent_id;
    this._type = _type;
  }
  target_id: string;
  parent_id: string;
  _type: string;
}
